src/Skills/index.js

```diff
@@ -66,8 +66,8 @@ export const Skills = ({ queryDetails }) => {
           }
         />
         <div>
-          <Chip label={skill.gradeId} className={classes.gradeChip} />
-          <Chip label={skill.categoryId} className={classes.categoryChip} />
+          <Chip label={skill.grade.title} className={classes.gradeChip} />
```

Comment:
Think we might have to do some transformations on this. I tend to use lodash https://lodash.com/docs/4.17.15#startCase

Reply:
Ah that's from LFH-55 🤦‍♀️ will change there